Tests/MapboxMapsTests/MapView/Unit Tests/MapInitOptionsTests.swift

```diff
@@ -0,0 +1,142 @@
+import XCTest
+@testable import MapboxMaps
+
+class MapInitOptionsTests: XCTestCase {
+
+    private var dataSourceReturnValue: Any = false
+    private var oldDefaultAccessToken: String = ""
+
+    override func setUp() {
+        super.setUp()
+        oldDefaultAccessToken = CredentialsManager.default.accessToken
+    }
+    override func tearDown() {
+        super.tearDown()
+        CredentialsManager.default.accessToken = oldDefaultAccessToken
+
+        oldDefaultAccessToken = ""
+        dataSourceReturnValue = false
+    }
+
+    func testDefaultMapInitOptionsAreOverridden() {
+        do {
```

Comment:
What's the purpose of these `do` blocks?

Reply:
just a scope so you can use the same variable name?

Reply:
maybe just say `var updatedMapInitOptions`?

Reply:
Just convenient scoping.